RELEASES.md

```diff
@@ -0,0 +1,42 @@
+# Releases
+
+The Tendermint specification follows [semantic versioning](https://semver.org/). Implementations of the Tendermint protocol are encouraged to either mirror versions or have some form of mapping for clear readibility of the state of the implementation. The `master` branch is used for active protocol development. Developers should be aware that the specifications are subject to change and should refer to tagged releases rather than head. In the event of a security vulnerability, fixes may be backported to prior major releases, but generally only the last major release is supported. The following documentation is provided to describe the release process.
+
+Releases capture a complete snapshot of the specification at a point in time. The protocols described in a release must be coherent and capable of functioning together. A release must include no drafts or outstanding/unanswered comments. For specifications that contain references to versions used for formal verification, there must be no ambiguity around what version is being used. Uncompleted or proposed but not yet finalized work can remain in master but it is preferable that large protocol designs use a separate long-lived development branch.
+
+## Major Releases
+
+Before any major release, it is preferable to create release candidates (RC) for Tendermint implementations as well as applications to test out and validate. This is done using git tags which are built off major release branches i.e. `v0.35.x`.
+
+Tags for RCs should follow the "standard" release naming conventions, with `-rcX` at the end (for example, `v0.35.0-rc0`) and incrementing from `0`.
+
+If this is the first RC, then you'll need to create a new branch for the major release:
+
+1. Start on `master`
+2. Create and push the new branch: `git checkout -b v0.35.x && git push origin v0.35.x`
+
+Now, to prepare the new RC:
```

Comment:
I'm guessing this was copy+pasted from Tendermint? I'm not really sure we need to actually do release candidates of the spec. RCs AFAICT exist to get the software into users hands and indicate a certain level of stability so that would-be early adopters can begin integrating. I don't see an analog for the spec so I'm not sure we need an analogous mechanism.